qiskit_nature/second_q/transformers/__init__.py

```diff
@@ -16,12 +16,6 @@
 
 .. currentmodule:: qiskit_nature.second_q.transformers
 
-Transformers act on a :class:`~qiskit_nature.drivers.QMolecule` to produce an altered copy of it as
```

Comment:
Removing this entirely is fine for now. These docs will need to be rewritten once the ongoing refactoring process is done anyways :+1: